contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -424,7 +424,8 @@ CREATE VIEW information_schema_tsql.tables AS
 		AND (pg_has_role(c.relowner, 'USAGE')
```

Comment:
Add standard PR template. Also, please add relevant reviewer with information schema domain knowledge. Looks good in general and upgrade perspective.